agent/ebs/watcher_test.go

```diff
@@ -65,6 +68,9 @@ func TestHandleEBSAttachmentHappyCase(t *testing.T) {
 	ctx := context.Background()
 	taskEngineState := dockerstate.NewTaskEngineState()
 	mockDiscoveryClient := mock_ebs_discovery.NewMockEBSDiscovery(mockCtrl)
+	mockTaskEngine := mock_engine.NewMockTaskEngine(mockCtrl)
+	mockTaskEngine.EXPECT().GetEbsDaemonTask().Return(nil).AnyTimes()
+	mockTaskEngine.EXPECT().GetDaemonManagers().Return(nil).AnyTimes()
```

Comment:
nit: how about keeping expectations stricter by specifying the exact number of calls expected instead of `AnyTimes`?